bindings/tox.ini

```diff
@@ -9,7 +9,8 @@ requires = tox-pyo3
 pyo3 = True
 deps =
     pytest
-commands =
-    pytest -vvvv
+    pytest-timeout
+    aiounittest
+commands = pytest {posargs:tests} --timeout=300 -vvvvv
```

Comment:
5 v's?

Reply:
this increases the verbosity level of the pytest.